Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -25,10 +25,12 @@
 
 enum AspectMode
 {
-	ASPECT_AUTO       = 0,
-	ASPECT_FORCE_16_9 = 1,
-	ASPECT_FORCE_4_3  = 2,
-	ASPECT_STRETCH    = 3,
+	ASPECT_AUTO        = 0,
+	ASPECT_FORCE_16_9  = 1,
+	ASPECT_FORCE_4_3   = 2,
+	ASPECT_STRETCH     = 3,
+	ASPECT_ANALOG      = 4,
+	ASPECT_ANALOG_WIDE = 5,
```

Comment:
Why keep Aspect Force 16:9/4:3, they produce incorrect results and there is no good reason for users to pick them.

Perhaps add a "Force 1:1 Pixels" or "Disable Stretching" option for users who want to take screenshots of the raw EFB output.